pt/lessons/basics/strings.md

```diff
@@ -29,14 +29,23 @@ Internamente, as strings em Elixir são representadas como uma sequência de byt
 Qual a diferença? Cada valor em uma lista de caracteres corresponde ao número Unicode do caracter, enquanto em um binário os valores são codificados em UTF-8. Vamos ver isso mais a fundo:
 
 ```elixir
-iex(5)> 'hełło'
+iex> 'hełło'
 [104, 101, 322, 322, 111]
-iex(6)> "hełło" <> <<0>>
+iex> "hełło" <> <<0>>
 <<104, 101, 197, 130, 197, 130, 111, 0>>
 ```
 
 `322` é o número Unicode de ł, representado em UTF-8 pelos dois bytes `197`, `130`.
 
+Você pode obter o codepoint de um carácter usando `?`
```

Comment:
There is a wrong accent in "caracter". I believe "caractere" would fit better here.

Reply:
@anderkonzen thanks! I fixed the typo. Could you please take another look?